src/Repository/CommitteeRepository.php

```diff
@@ -358,6 +358,80 @@ public function countApprovedForReferent(Adherent $referent): int
         ;
     }
 
+    public function findApprovedForReferentAutocomplete(Adherent $referent, $value): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        $qb = $this->createQueryBuilder('committee')
+            ->select('committee.uuid, committee.name')
+            ->join('committee.referentTags', 'tag')
+            ->where('committee.status = :status')
+            ->andWhere('committee.name LIKE :word')
+            ->setParameter('word', $value.'%')
+            ->setParameter('status', Committee::APPROVED)
+            ->orderBy('committee.name')
+        ;
+
+        $tagsFilter = $qb->expr()->orX();
+
+        foreach ($referent->getManagedArea()->getTags() as $key => $tag) {
+            $tagsFilter->add("tag.code IN (:tag_$key)");
```

Comment:
Could use `=` instead of `IN` 
And could use entity directly instead of the `$tag->getCode()`.

Reply:
I think the tag condition can be put outside of the foreach no ?

```php
$tagsFilter
    ->add("tag IN (:tags)")
    ->setParameter('tags', $referent->getManagedArea()->getTags())
;
```